Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1283,10 +1556,13 @@ def build_one_value_parsed_output(**kwargs) -> Tuple[Dict[str, Any], str]:
         TypeError: will be raised by  parse_two_keys_dict response is not a dictionary
     """
     command_parse_and_output_data = kwargs['command_parse_and_output_data']
+    collection_name = command_parse_and_output_data.get('collection_name')
     response = kwargs['handled_result']
-    single_val_dict = parse_two_keys_dict(response)
-    readable_output = tableToMarkdown(name=command_parse_and_output_data['table_name'], t=single_val_dict)
-    return single_val_dict, readable_output
+    output = parse_two_keys_dict(generate_list_dicts(response[collection_name]))
+    output['DATETIME'] = response.get('DATETIME')
+    output['TEXT'] = response.get('TEXT')
```

Comment:
Well, Danil asked me to add Text and DATETIME to mine, I thought it would be better to make sure all the commands with DATETIME and TEXT as well. We can take it offline with DAnil

Reply:
Ok, although we should be sure that all those commands that use `build_one_value_parsed_output`  really have a text and datetime.

Other than that, there are the rest of the commands, that probably also have datetime and text fields but it was not requested that they would have datetime and text as output. I think that it should be consistent with all of Qualys commands. So we should actually talk with Danil about it.